DESCRIPTION

```diff
@@ -9,6 +9,7 @@ Authors@R: c(
       person("Hadley", "Wickham", role = c("ctb")),
       person("jQuery contributors", role = c("ctb", "cph"), comment = "jQuery library"),
       person("Leaflet contributors", role = c("ctb", "cph"), comment = "Leaflet library"),
+      person("Leaflet Providers contributors", role = c("ctb", "cph"), comment = "Leaflet Providers plugin"),
```

Comment:
I think that they should be listed as `ctb` only. See the LICENSE:
  https://github.com/Leaflet/Leaflet/blob/master/LICENSE